wodles/aws/aws_s3.py

```diff
@@ -2990,7 +2995,13 @@ def get_alerts(self):
                     token = None
 
                     if db_values:
-                        if db_values['start_time'] and db_values['start_time'] > start_time:
+                        if self.reparse: 
+                            debug('Reparse mode ON', 1)
```

Comment:
I'd place this message outside of the `if db_values` block. It could be confusing if you execute the module for the first time and you don't get any message related to the reparse mode, but then you execute it again and get it.